### PR TITLE
Enhancement: do not show the bribe directly when hailing a hostile ship

### PR DESCRIPTION
**Pull request by Hurleveur**
**Feature:** This PR implements the feature request detailed and discussed in issue #7915

## Feature Details
When hailing a hostile ship they will first have their normal phrase, and clicking on offer bribe the first time will make them give their price; whilst the second time will make them take your money, if they want to ofc.

## UI Screenshots
Bribing an Elenchus pirate.

## Usage Examples
Pirates should talk to you normally instead of only caring bout money (greedy buggers)

## Testing Done
~~not tested yet~~
tested, works as expected

### Automated Tests Added
n/a

## Performance Impact
n/a

### CODE REVIEW

**Comment by warp-core** [comment]
I don't think UI screenshots are not applicable here.
Is the button still labelled `Offer bribe` after the first click?
Since we've already offered the bribe, it should probably change to `Pay bribe` or similar.

**Comment by Hurleveur** [comment]
It keeps being offer bribe. I was thinking that as well ~~but I got lazy and it didnt seem like a requirement~~
I'll look into it

**Comment by Hurleveur** [comment]
I have no idea whats wrong with this test, or how this PR could affect it.

**Comment by Hurleveur** [comment]
> I don't think UI screenshots are not applicable here. Is the button still labelled `Offer bribe` after the first click? Since we've already offered the bribe, it should probably change to `Pay bribe` or similar.

Feedback addressed.

**Review by McloughlinGuy — APPROVED** [review]
This looks good, especially now that there's a pay bribe button. Originally I thought saying "yes/no" would be better, but you have to exit out of the menu and open it again to see a new hail anyways and it's really easy to do so on second thought that doesn't really make sense. So all in all, this looks great, thanks for doing this!

File: source/HailPanel.cpp [hunk 1]
```diff
@@ -61,17 +61,8 @@ HailPanel::HailPanel(PlayerInfo &player, const shared_ptr<Ship> &ship, function<
 		message = "(There is no response to your hail.)";
 	else if(!hasLanguage)
 		message = "(An alien voice says something in a language you do not recognize.)";
-	// Update default hail responses based on the hailed ship's government and status condition.
 	else if(gov->IsEnemy())
-	{
-		if(!ship->IsDisabled())
```
Comment (quyykk) [inline source/HailPanel.cpp:67]: Since you removed this check you can now bribe disabled ships. You should keep it imo

**Comment by Amazinite** [comment]
On the third image, the second button is blank after paying the bribe. What's the current behavior when bribing?

**Comment by warp-core** [comment]
image

**Comment by Amazinite** [comment]
Then paying a bribe should result in the button having grayed-out "Pay Bribe" text.

**Comment by Hurleveur** [comment]
Booleans are a pain to me, how to do that with can bribe, offered bribe and the boolean logic we have whilst making sure both arent shown at the same time at any given point?

**Comment by quyykk** [comment]
Just change the `active if "offered bribe"` to `active` I think

**Comment by Hurleveur** [comment]
I made it show a greyed offer bribe text instead of pay bribe bc that was easier to do codewise (I hope thats what I did anyway since I'm coding from phone)

**Comment by quyykk** [comment]
yeah it shows the disabled "offer bribe" button

File: data/interfaces.txt [hunk 2]
```diff
@@ -1234,10 +1234,16 @@ interface "hail panel"
 		center 250 115
 		dimensions 80 30
 
+	visible if "!hide bribe offer"
 	active if "can bribe"
 	button b "Offer _Bribe"
 		center 130 115
 		dimensions 140 30
+	visible if "offered bribe"
```
Comment (Hurleveur) [inline data/interfaces.txt:1242]:
```suggestion
	visible if "hide bribe offer"
```

File: data/interfaces.txt [hunk 3]
```diff
@@ -1234,10 +1234,16 @@ interface "hail panel"
 		center 250 115
 		dimensions 80 30
 
+	visible if "!hide bribe offer"
```
Comment (Hurleveur) [inline data/interfaces.txt:1237]:
```suggestion
	visible if "!offered bribe"
```

**Comment by Hurleveur** [comment]
Here we go, it now works as per Amazinite's request

File: source/HailPanel.cpp [hunk 4]
```diff
@@ -61,17 +61,8 @@ HailPanel::HailPanel(PlayerInfo &player, const shared_ptr<Ship> &ship, function<
 		message = "(There is no response to your hail.)";
 	else if(!hasLanguage)
 		message = "(An alien voice says something in a language you do not recognize.)";
-	// Update default hail responses based on the hailed ship's government and status condition.
 	else if(gov->IsEnemy())
```
Comment (Hurleveur) [inline source/HailPanel.cpp:64]:
```suggestion
	else if(gov->IsEnemy() && !ship->IsDisabled())
```

File: source/HailPanel.cpp [hunk 5]
```diff
@@ -61,17 +61,8 @@ HailPanel::HailPanel(PlayerInfo &player, const shared_ptr<Ship> &ship, function<
...
-		if(!ship->IsDisabled())
-		{
-			SetBribe(gov->GetBribeFraction());
-			if(bribe)
-				message = "If you want us to leave you alone, it'll cost you "
-					+ Format::Credits(bribe) + " credits.";
-		}
-	}
+	else if(gov->IsEnemy() && !ship->IsDisabled())
+		SetBribe(ship->GetGovernment()->GetBribeFraction());
```
Comment (Amazinite) [inline source/HailPanel.cpp:65]:
```suggestion
		SetBribe(gov->GetBribeFraction());
```

**Pull request merged**